gradle.properties

```diff
@@ -2,14 +2,15 @@
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
+org.gradle.java.installations.fromEnv=JAVA_17_HOME
```

Comment:
What is this needed for?

Reply:
ah, hm
that one was because that's the environment variable containing java17's install location on my machine and cradle needs to be told where it is to be able to use it, but it shouldn't break anyone else's workflow?

Reply:
can't you just set JAVA_HOME? (IntelliJ should do that for you, if configured in the project and gradle settings)